### DISCUSSION TITLE
Internal bosch motion sensor support

### DISCUSSION THREAD

**Discussion by eugenov** [r57zone/DualShock4-emulator#71] [Q&A] [answered]
Hello!
I have gpd winmax2 with bosch gyro sensor. How to make it work with this utility? Only external android/button emulated gyro supported?
Thanks!

**Comment by r57zone** [comment, marked as answer]
Hello, only Android gyroscope and button are supported, others are not supported unfortunately.

**Reply by eugenov** [reply]
Hmmm is there any chance this feature will appear?

**Reply by r57zone** [reply]
unfortunately no chance
